test/unit/integrations/front.spec.ts

```diff
@@ -1,5 +1,7 @@
 import { defaultEnvironment } from '@balena/jellyfish-environment';
-import * as errors from '@balena/jellyfish-sync/build/errors';
+import { defaultPlugin } from '@balena/jellyfish-plugin-default';
+import { PluginManager } from '@balena/jellyfish-worker';
+import { errors as syncErrors } from '@balena/jellyfish-worker/build/sync';
```

Comment:
Deep import. Gotta either export `syncErrors` from the worker or merge the two error sets